webapp/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Container, Grid, Paper } from "@mui/material";
+import { MatchHistory } from "./components/match-history";
+import Copyright from "../Utils/Copyright";
+import Choose from "./components/Choose";
+import Connected from "./components/Connected";
+import { mdTheme } from "../Utils/Dashboard";
+import NewAppBar from "../Utils/NewAppBar";
+
+const Home = () => (
+
+    <>
+        <NewAppBar>
+        </NewAppBar>
```

Comment:
@automaths en general, ce n'est pas comme cela qu'on inclus un component, mais plutot comme ceci:

```suggestion
        <NewAppBar/>
```